src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Image/Image.ts

```diff
@@ -122,20 +136,41 @@ export var showImageDirective = (
             cssClass: "@",
             alt: "@?",
             format: "@?", // defaults to "detail"
-            imageMetadataNick: "@?" // defaults to SIImageMetadata.nick
+            imageMetadataNick: "@?", // defaults to SIImageMetadata.nick
+            fallbackUrl: "@?", // defaults to "/static/fallback_$format.jpg";
+            isImageMissing: "=?"
         },
         link: (scope) => {
-            if ( ! scope.imageMetadataNick) { scope.imageMetadataNick = SIImageMetadata.nick; }
-            if ( ! scope.format) { scope.format = "detail"; }
-
-            var fallbackUrl = "/static/fallback_" + scope.format + ".jpg";
+            var imageMetadataNick = () =>
+                scope.imageMetadataNick ? scope.imageMetadataNick : SIImageMetadata.nick;
+            var format = () => scope.format ? scope.format : "detail";
```

Comment:
I understand why you used a function. But `scope.format || "detail"` is shorter and less redundant than `scope.format ? scope.format : "detail"`.